### DISCUSSION TITLE
Login after each refresh seems to be back

### DISCUSSION THREAD

**Discussion by TonySimonovsky** [danny-avila/LibreChat#3926] [Troubleshooting]
I just updated LibreChat to the newest version (thanks for the artifacts!) and the login after every refresh problem is back.

The solution in https://www.librechat.ai/blog/2024-05-16_unsecured_http doesn't work anymore - I have the required lines in override file, but still am required to login after each page refresh.

**Comment by sydnorlabs** [comment]
I also have this issue

**Comment by TonySimonovsky** [comment]
Here's how I solved the problem.

In general, you need an SSL certificate on your server.

Here's how I did it:

https://github.com/TonySimonovsky/aichamp_code_snippets/tree/main/librechat_digitalocean_login_after_refresh_issue_fix
